Source/WebKit/ModelProcess/cocoa/ModelProcessModelPlayerProxy.mm

```diff
@@ -390,9 +390,9 @@ static RESRT computeSRT(CALayer *layer, simd_float3 originalBoundingBoxExtents,
         boundingBoxCenter = srt.scale * originalBoundingBoxCenter;
```

Comment:
Do we still need boundingBoxCenter for this function? (and other variables computed above)